app/javascript/controllers/edit_deposit_controller.js

```diff
@@ -42,12 +43,27 @@ export default class extends Controller {
   parseErrors(data) {
     for (const [fieldName, errorList] of Object.entries(data)) {
       const key = `${fieldName}Field`
-      const target = this.targets.find(key)
-      if (target)
-        target.dispatchEvent(new CustomEvent('error', { detail: errorList }))
-      else
-        console.error(`unable to find target for ${key}`)
+      this.dispatchError(key, errorList)
     }
     window.scrollTo(0, 80)
   }
+
+  clearErrors() {
+    const clearErrorsTargets = ["titleField", "fileField",
+                                "keywordsField", "contributorsField",
+                                "embargo-dateField"];
+
+    for (const key of clearErrorsTargets) {
+      this.dispatchError(key, [])
+    }
+  }
+
+  dispatchError(key, errorList) {
+    const target = this.targets.find(key)
+    if (target)
+      target.dispatchEvent(new CustomEvent('error', { detail: errorList }))
```

Comment:
So passing an empty list to `errorList` clears the errors? Huh, OK!

Reply:
yeah, that allowed me to use the existing error methods to automatically clear error messages.  it is a bit confusing in that it's an error event, but also sending an empty error list doesn't seem like a terrible way to signal "no errors", so 🤷.  i wasn't wild about this, but i didn't give it a ton of consideration, and am happy to do something with more informative method naming if you'd rather.

Reply:
if it works, it works!